src/main.ts

```diff
@@ -219,13 +221,21 @@ export async function main(args: string[]): Promise<number> {
           break
         }
         case "gcc":
-        case "mingw":
         case "cygwin":
         case "msys": {
           const installationInfo = await setupGcc(getVersion("gcc", version, osVersion), join(setupCppDir, "gcc"), arch)
           successMessages.push(getSuccessMessage("gcc", installationInfo))
           break
         }
+        case "mingw": {
+          const installationInfo = await setupMingw(
+            getVersion("mingw", version, osVersion),
+            join(setupCppDir, "gcc"),
```

Comment:
Should this become mingw?

```suggestion
            join(setupCppDir, "mingw"),
```